src/components/contexts/AuthContext.js

```diff
@@ -4,29 +4,32 @@ import {
   signInWithEmailAndPassword,
   onAuthStateChanged,
   getAuth,
+  connectAuthEmulator,
 } from "firebase/auth";
 import app from "../../firebase";
 
 const AuthContext = React.createContext();
 
-export function useAuth() {

```

Comment:
here, export function useAuth() {}vs export const useAuth = () =>{}

which one is better?

Also, export const useAuth = () =>{} works, 
but const useAuth = () =>{} 
and export default useAuth (the code will not work, why?)